.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: Gsutil CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [pypy2, 2.7, pypy3, 3.5, 3.6, 3.7]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        set -x
+        pip install .
+        git submodule update --init --recursive
+    - name: Lint
+      if: ${{ matrix.python-version  == 3.6 }}
+      run: |
+        set -x
+        pip install pylint
+        pip install yapf
+        test/run_pylint.sh;
```

Comment:
Why do these require semi-colon in the end, but other commands don't?

Reply:
Good catch